.github/ISSUE_TEMPLATE/support_powertools.yml

```diff
@@ -0,0 +1,64 @@
+name: Support Lambda Powertools (become a reference)
+description: Add your organization's name or logo to the Lambda Powertools documentation
+title: "[Support Lambda Powertools]: <your organization name>"
+labels: ["customer-reference"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for becoming a reference customer. Your support means a lot to us. It also helps new customers to know who's using it.
+
+        If you would like us to also display your organization's logo, please share a link in the `Company logo` field.
+  - type: input
+    id: organization
+    attributes:
+      label: Organization Name
+      description: Please share the name of your organization
+      placeholder: ACME
+    validations:
+      required: true
+  - type: input
+    id: name
+    attributes:
+      label: Your Name
+      description: Please share your name
+    validations:
+      required: true
+  - type: input
+    id: job
+    attributes:
+      label: Your current position
+      description: Please share your current position at your company
+    validations:
+      required: true
+  - type: input
+    id: logo
+    attributes:
+      label: (Optional) Company logo
+      description: Company logo you want us to display. You also allow us to resize for optimal placement in the documentation.
```

Comment:
(Optional) Shall we specify the preferred size here to reduce rework? Or at least a link to an example page?

Reply:
We can't (legal) on size otherwise every company will ask for a different one (visibility, competitors etc).

On example page - that's what we're creating now